### ISSUE TITLE
Distance to Wall

### ISSUE THREAD

**Issue by stlgolfer** [MICDSRobotics-9911/Rampage-SkyStone2019-2020#21]
# Overview
On both of our autonomii, we literally slam into the wall to make get to where we need to be. This is not the most time-effective way to do this.

# Solution
1) use encoders
2) put a color sensor on the bottom (use the one that's currently in the front) to figure out where the tape is, then use encoders to take us to the wall (this will account for the varying distances when we pick up the skystone). **Be advised**: we will need to make sure the logic for this algorithm will fit with #19

**Comment by lmccarty5** [comment]
Couldn't we just decrease the velocity and add a touch sensor on either side?

**Comment by stlgolfer** [comment]
Yes, we could, although the color sensor would fall outside the 18x18.

**Comment by lmccarty5** [comment]
We could probably fix that mechanically. We would just need to cut out a bit of plexi.
